lib/takeaway.rb

```diff
@@ -0,0 +1,76 @@
+require 'twilio-ruby'
+
+class TakeAway
+
+  attr_reader :menu
+  attr_accessor :order, :prices
+
+  def initialize
+    @order = {}
+    @prices = []
+  end
+
+  def add_menu(menu)
+    @menu = menu.dishes
+  end
+
+  def show_menu
+    menu
+  end
+
+  def place_order(items = {})
```

Comment:
separate items from quantity so you can add to the quantity, without having to add the same item twice